packages/lib-classifier/src/store/AnnotationsStore.js

```diff
@@ -17,6 +17,20 @@ const AnnotationsStore = types
     }
   }))
   .actions(self => {
+    function afterAttach() {
+      createClassificationObserver()
+    }
+
+    function createClassificationObserver() {
+      const classificationsDisposer = autorun(() => {
+        const classifications = getParent(self)
```

Comment:
The name here is confusing because marks are also instances of `AnnotationsStore`.

`getParent(self)` returns the map for a member of a map, so this will return `classifications.resources` for a classification and `tool.marks` for a mark.

Reply:
That was a mistake on my part in reading the code. This observer can be more generalized then.